docs/installation.rst

```diff
@@ -224,3 +230,52 @@ If you want to make changes in the code, set up the development mode:
 
 pyqg is a work in progress, and we really encourage users to contribute to its
 :doc:`/development`
+
+
+.. _advanced-install:
+
+Installing with OpenMP support on OSX
+-------------------------------------
+
+There are two options for installing on OSX with OpenMP support.  Both methods require using the Anaconda distribution of
+Python.
+
```

Comment:
On my iMac, the only way I have successfully managed to compile with openmp is with the anaconda distribution.  As mentioned in #100, when I use builtin python Cython tries to pass `-Wshorten-64-to-32` to the compiler as it is expecting `clang`, but this is not a valid flag for HPC gcc.

We can change the line requiring anaconda, it may be an issue just affecting me.